components/script/resources/media-controls.js

```diff
@@ -62,6 +62,7 @@
 
   function formatTime(time, showHours = false) {
     // Format the duration as "h:mm:ss" or "m:ss"
+    if (isNaN(time) || !isFinite(time)) return "00:00";
```

Comment:
```suggestion
    if (isNaN(time) || !isFinite(time)) {
        return "00:00";
    }
```

Also you have put this new code in between the comment and the line of code that is ensuring that the formatting is correct, so I would move it above the comment.